README.md

```diff
@@ -126,6 +186,8 @@ The following variables are used to configure the Hydrolix connection. These var
 
 #### Required Variables
 * `HYDROLIX_HOST`: The hostname of your Hydrolix server
+* `HYDROLIX_TOKEN`: The Hydrolix service account token
+  ##### OR
 * `HYDROLIX_USER`: The username for authentication
 * `HYDROLIX_PASSWORD`: The password for authentication
```

Comment:
```suggestion
* `HYDROLIX_TOKEN`: The Hydrolix service account token (omit if using username/password)
* `HYDROLIX_USER`: The username for authentication (omit if using service account)
* `HYDROLIX_PASSWORD`: The password for authentication (omit if using service account)
```
The `OR` may be read as dropping `HYDROLIX_HOST` as well -- at the least, we shouldn't use header syntax for the `OR`